src/components/Card/Card.test.tsx

```diff
@@ -11,23 +11,23 @@ const itemWithImage = { title: 'This is a movie', duration: 120, cardImage: 'htt
 
 describe('<Card>', () => {
   it('renders card with video title', () => {
-    const { getByText } = renderWithRouter(<Card item={item} onClick={() => ''} />);
+    const { getByText } = renderWithRouter(<Card item={item} onClick={() => ''} url="https://test.dummy.jwplayer.com" />);
```

Comment:
Whoops, there aren't any snapshot tests to actually check the url part of this. Can you add one for the Card component?